### ISSUE TITLE
Save individual user zip codes

### ISSUE THREAD

**Issue by ghost** [myano/jenni#54]
Idea is for users to have jenni store user zip codes. 
Example: 

KindOne: .set weather 31216

// so that later on, when i do 

KindOne: .weather

// I get 

jenni: Clear ☼, 50.0℉ (10℃), 30.00in (1013mb), Calm 0kt (↑) - KMCN 08:53Z

//Without having to specify my zipcode.

**Comment by kaneda** [comment]
I don't think this is feasible without https://github.com/myano/jenni/issues/107 finished first.

**Comment by myano** [comment]
Agreed, this will need #107 to be done in a sane way.
